_tour/tuples.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: tour
+title: Tuples
+
+discourse: true
+
+partof: scala-tour
+
+num: 
+next-page: mixin-class-composition
+previous-page: traits
+topics: tuples
+
+redirect_from: "/tutorials/tour/tuples.html"
+---
+
+In Scala, a tuple is a class that can hold elements of different types.
+Tuples are immutable.
+
+Tuples come handy when we have to return multiple values from a function.
+
+A tuple can be created as:
+
+```tut
+val ingredient = ("Sugar" , 25)
+
+println(ingredient.getClass) // class scala.Tuple2 
```

Comment:
Not sure if we should expose beginners to `.getClass` here.

Reply:
agree, the better way to demonstrate that something conforms to a certain type is to show that it still compiles if you add a type ascription such as `(String, Int)` or `Tuple2[String, Int]`